clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -410,7 +410,10 @@ fatbinary(ArrayRef<std::pair<StringRef, StringRef>> InputFiles,
     Targets.push_back(Saver.save("hipv4-amdgcn-amd-amdhsa--" + Arch));
   CmdArgs.push_back(Saver.save(llvm::join(Targets, ",")));
 
-  CmdArgs.push_back("-input=/dev/null");
+  if (Triple.isOSWindows())
```

Comment:
This does not depend on the target platform, but the host OS that runs clang-linker-wrapper (`#ifdef _WIN32`)

Reply:
Right, thanks